apps/judicial-system/web/src/components/Header/Header.spec.tsx

```diff
@@ -0,0 +1,45 @@
+import { rest } from 'msw'
+import { setupServer } from 'msw/node'
+import { act, render, screen } from '@testing-library/react'
+
+import { IntlProviderWrapper } from '../../utils/testHelpers'
+import { Header as HeaderContainer } from '..'
+
+jest.mock('next/router', () => ({
+  useRouter() {
+    return {
+      pathname: '',
+    }
+  },
+}))
+
+const server = setupServer(
+  rest.get('/api/geoLocation/getCountryCode', (_req, res, ctx) => {
+    return res(ctx.status(200), ctx.json({ countryCode: 'IS' }))
+  }),
+)
+
+beforeAll(() => server.listen())
+afterEach(() => server.resetHandlers())
+afterAll(() => server.close())
+
+describe('Header', () => {
+  it('should not render a "ask for help" button if the user is in Iceland', async () => {
```

Comment:
~ Not part of my codeowner files but I think the test description is misleading? it should only render the button for icelandic users?

Reply:
Yea, there was a specific reason for this but I was unable to properly write this test so I removed it..